docker-compose.yml

```diff
@@ -75,12 +44,9 @@ services:
       FUSIONAUTH_APP_MEMORY: ${FUSIONAUTH_APP_MEMORY}
       FUSIONAUTH_APP_RUNTIME_MODE: ${FUSIONAUTH_APP_RUNTIME_MODE}
       FUSIONAUTH_APP_URL: http://fusionauth:9011
-      SEARCH_SERVERS: http://search:9200
-      SEARCH_TYPE: elasticsearch
```

Comment:
You need to set the `SEARCH_TYPE` to `database`, I believe. Looks like the default is `database` but better to be explicit.